Forms/GridField/GridField.php

```diff
@@ -307,12 +307,12 @@ public function getState($getData = true) {
 	 */
 	public function FieldHolder($properties = array()) {
 		Requirements::css(THIRDPARTY_DIR . '/jquery-ui-themes/smoothness/jquery-ui.css');
-		Requirements::css(FRAMEWORK_DIR . '/client/dist/styles/GridField.css');
+		Requirements::css(trim(FRAMEWORK_DIR . '/client/dist/styles/GridField.css', '/'));
```

Comment:
What about introducing a convention of the constants having a trailing slash?

PS: `ltrim`?